code/src/lib/Image.js

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+
+export const TrashBin = styled.img `
```

Comment:
Maybe you could of added ` cursor: pointer;` here. Then it would make it clear to the user that this icon is clickable, and therefore avoiding any potential accidental deletions.

Reply:
Thanks Jamie, so true, this is now added! =)